open-api/rest-catalog-open-api.py

```diff
@@ -1311,11 +1327,19 @@ class LoadViewResult(BaseModel):
 
     - `token`: Authorization bearer token to use for view requests if OAuth2 security is enabled
 
+    ## Storage Credentials
+
+    Credentials for ADLS / GCS / S3 / ... are provided through the `storage-credentials` field.
+    Clients should first check whether the respective credentials exist in the `storage-credentials` field before checking the `config` for credentials.
```

Comment:
Nit: Should we replace the use of "should" with "must" throughout this?

Reply:
I'm fine either way and it seems we have a bunch of other places that use `should` where `must` could be used as well, so I updated this to `must`